src/modules/hip/kernel/mel_filter_bank.hpp

```diff
@@ -0,0 +1,164 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__device__ __forceinline__ float calcMel(float *input, int melBin,
```

Comment:
Please change naming convention of input to srcPtr

Reply:
Done